_component_design/key-value-pairs.md

```diff
@@ -9,32 +9,32 @@ preview-image: preview-images/key-value.svg
 resource: true
 status: stable
 last-modified: 2017-11-07
```

Comment:
Ditto

Reply:
Added YAML code back in; updated last-modified.